jbehave-support-core/src/main/resources/report.xslt

```diff
@@ -13,6 +13,12 @@
 
     <xsl:output method="html" indent="yes"/>
 
+    <xsl:variable name="smallcase" select="'abcdefghijklmnopqrstuvwxyz'"/>
```

Comment:
Do we need smallcase, uppercase and totalStories here? And I guess this will probably just overwrite index-source that is in envelope - I think that ideally we should have it in one place only a pass it as a parameter or something if possible.